src/nanodbc.cpp

```diff
@@ -1681,6 +1683,15 @@ class statement::statement_impl
         bind(param, values, elements, direction);
     }
 
+    // handles multiple string values
+    void bind_strings(
+        short param,
+        const std::vector<string_type>& values,
+        std::size_t elements,
```

Comment:
Since the vector carries size, the `elements` seem redundant. Would it be possible to get rid of it?